server/src/containers/docker.js

```diff
@@ -13,23 +12,24 @@ const containerOptions = {
   StdinOnce: false,
   HostConfig: {
     NetworkMode: config.dockerNetwork,
-    Binds:
-      !config.isProduction && config.hostPwd
-        ? [
-            `${config.hostPwd}/quests/helloworld/home:/home/commander`,
-            `${config.hostPwd}/packages/container:/usr/local/lib/container`,
-          ]
-        : [],
   },
 }
 
 // const network = engine.getNetwork(config.dockerNetwork)
 
-export function createContainer(name) {
-  return engine.createContainer({
+export function createContainer(name, tag) {
+  const option = {
     ...containerOptions,
     name,
-  })
+    Image: `${config.dockerImage}:${tag}`,
+  }
+  if (!config.isProduction && config.hostPwd && config.mountQuest) {
+    option.HostConfig.Bind = [
+      `${config.hostPwd}/quests/${config.mountQuest}/home:/home/commander`,
+      `${config.hostPwd}/packages/container:/usr/local/lib/container`,
+    ]
+  }
+  return engine.createContainer(option)
```

Comment:
Does this part mean only if production mode got hostPwd and mountQuest can create container automatically?

Reply:
No, this part is for mounting host directories into container, it's for development perpose. It will only mount when node env is set to development. When in production, each container is fully isolated.

Reply:
Oh yeah I misread it with the exclamation